src/Blocks/BlockOre.h

```diff
@@ -33,16 +33,16 @@ class cBlockOreHandler:
 
 		auto & Random = GetRandomProvider();
```

Comment:
These changes are all good improvements to the readability, thank you. This line can be moved down a bit now too.